module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1199,6 +1199,22 @@ public function getFilterList($excludeCheckboxFilters = false)
         return $list;
     }
 
+    /**
+     * Returns only the exclude filters (field starting with '-').
+     *
+     * @return array an array field => value without the '-' for the field
+     */
+    public function getExcludeFilters()
+    {
+        $result = [];
+        foreach ($this->filterList as $field => $values) {
+            if (str_starts_with($field, '-')) {
+                $result[substr($field, 1)] = $values;
+            }
```

Comment:
I would suggest using the existing parsing method here; this way, if other backends use different syntax, the functionality will work consistently.

```suggestion
            [$operator, $fieldName] = $this->parseOperatorAndFieldName($field);
            if ('NOT' === $operator) {
                $result[$fieldName] = $values;
            }
```